WooCommerce/Classes/ViewModels/ProductDetailsViewModel.swift

```diff
@@ -709,8 +709,10 @@ extension ProductDetailsViewModel {
             WebviewHelper.launch(product.externalURL, with: sender)
         case .productVariants:
             ServiceLocator.analytics.track(.productDetailViewVariationsTapped)
+            let isEditProductsRelease5Enabled = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProductsRelease5)
```

Comment:
Do you think it's worth it to inject `ServiceLocator.featureFlagService` in the init of this view model(and set it as a default parameter), so it's clear that this view model has that dependency?

Reply:
Sounds good 👍 done here 027119293fe8f2fc5f4d26f1a0adb66e9bdd34eb